deltachat-ios/Chat/Views/Cells/BaseMessageCell.swift

```diff
@@ -277,6 +277,20 @@ public class BaseMessageCell: UITableViewCell {
         if !msg.isInfo {
             bottomLabel.attributedText = getFormattedBottomLine(message: msg)
         }
+
+        if let quoteText = msg.quoteText {
+            quoteView.isHidden = false
+            quoteView.quote.text = quoteText
+
+            if let quoteMsg = msg.quoteMessage {
+                let contact = quoteMsg.fromContact
+                quoteView.senderTitle.text = contact.displayName
+                quoteView.senderTitle.textColor = contact.color
+            }
```

Comment:
as cells are reused, there should probably an `else { quoteView.senderTitle.isHidden = true }` or so. (just came over that, i know, it is WIP :)

Reply:
:+1:

Reply:
after having a second glance on it, I realized that sender title gets cleaned up in quoteView.prepareForReuse(). if the senderTitle.text is nil the sender title doesn't take any space